src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-link {
+  height: 40px;
+  width: 40px;
+}
+
+.header-nav {
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  line-height: 60px;
```

Comment:
if you set `height: 60px;` for `.nav-link` setting for `.nav-item` is not needed